app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -84,30 +85,32 @@ export default {
         ({ operator_address }) => operator_address in committedDelegations
       )
     },
-    unbondingTransactions: ({ allTransactions, delegation } = this) =>
-      // TODO still needed?
-      allTransactions
-        .filter(
-          transaction =>
-            // Checking the type of transaction
-            transaction.tx.value.msg[0].type === `cosmos-sdk/Undelegate` &&
-            // getting the unbonding time and checking if it has passed already
-            time.getUnbondingTime(
-              transaction,
-              delegation.unbondingDelegations
-            ) >= Date.now()
-        )
-        .map(transaction => ({
-          ...transaction,
-          unbondingDelegation:
-            delegation.unbondingDelegations[
-              transaction.tx.value.msg[0].value.validator_addr
-            ]
-        }))
-  },
-  mounted() {
-    this.$store.dispatch(`updateDelegates`)
-  }
+    unbondingTransactions: ({ transactions, delegation } = this) =>
+    // Checking the type of transaction
+    // getting the unbonding time and checking if it has passed already
+    /*eslint-disable */
+			transactions.staking &&
```

Comment:
looks like your tabs are set to 64 spaces

Reply:
but why disabling eslint?

Reply:
let's remove `eslint-disable`